.github/workflows/zeebe-ci.yml

```diff
@@ -8,7 +8,13 @@ on:
       - release-*
       - trying
       - staging
-  pull_request: { }
+    paths-ignore:
+      - '.github/workflows/operate-*'
+      - 'operate/**'
+  pull_request:
+    paths-ignore:
+      - '.github/workflows/operate-*'
+      - 'operate/**'
```

Comment:
I know you mentioned that you tried different approaches, but I'm wondering why was paths: zeebe/** not possible? Isn't that less to add?

We need to add here more to ignore if we add more components right? 🤔

Reply:
Alternative on the ignore to have `!zeebe`? 🤔

Reply:
> I know you mentioned that you tried different approaches, but I'm wondering why was paths: zeebe/** not possible? Isn't that less to add?

We also need to rebuild Zeebe in case something changes in the parent pom, in the bom, in the CI files, etc. The list of directories to include becomes rather long and we have to think of all edge cases that should trigger running the Zeebe CI.

> We need to add here more to ignore if we add more components right? 🤔

This is what I referred to with "Note that we still plan to unify Operate+Zeebe CI at some point so this mechanism will have to evolve over time -> its current implementation will not stay this way for long." but explained in a bit more detail in https://camunda.slack.com/archives/C06HTSPD5AP/p1709283337372289?thread_ts=1709221951.896589&cid=C06HTSPD5AP (since Roman asked as well).

Reply:
Might make sense to have multiple workflows at some point, for different purposes 🤔 

Like one which acts on the root, to build and test everything, when dependencies change. And then for each component a separate ci workflow ? 🤔